src/libs/README.md

```diff
@@ -0,0 +1,9 @@
+Jsoncons
+--------
```

Comment:
Add if the library is entirely copied here, or you pruned the unnecessary files.
Make sure that people know how to proceed in case they want to update the library (e.g., in case of security bugs are discovered, so we need to update the code).

Reply:
Done  :+1: